python/ray/serve/backend_worker.py

```diff
@@ -216,7 +196,7 @@ async def invoke_batch(self, request_item_list):
             return result_list
         except Exception as e:
             wrapped_exception = wrap_to_ray_error(e)
-            self.error_counter += batch_size
+            self.error_counter.add(batch_size)
```

Comment:
This doesn't match the description of the counter - "number of exceptions that have occurred in the backend." Should either only add one here or update the description to specify that it's the number of inputs that this has errored on.